src/System.Memory/tests/Span/SequenceCompareTo.int.cs

```diff
@@ -0,0 +1,145 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.SpanTests
+{
+    public static partial class SpanTests
+    {
+        [Fact]
+        public static void ZeroLengthSequenceCompareTo_Int()
+        {
+            var a = new int[3];
+
+            var first = new Span<int>(a, 1, 0);
+            var second = new ReadOnlySpan<int>(a, 2, 0);
+            int result = first.SequenceCompareTo<int>(second);
+            Assert.Equal(0, result);
+        }
+
+        [Fact]
+        public static void SameSpanSequenceCompareTo_Int()
+        {
+            int[] a = { 4, 5, 6 };
```

Comment:
> I just pushed the changes you requested regarding the test values.

We should make the same changes to the Span tests too (for int/long), just like ROSpan.

Reply:
Oh ! It seems I commited too quickly. It will be OK in a few minutes !